### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MirelaGeorgiou**
Added documentation for the Housing Get Accounts and Address API

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -443,4 +443,32 @@ A successful response should create the Work order in DRS and get the list of av
...
+      {
+        "housing_u_saff_rentacc": "228003480",
+        "aj_x002e_housing_post_code": "E9 6EJ",
+        "aj_x002e_housing_short_address": "5 Burnett Close"
+      }
+    ],
+    "apiStatusCode": 200
+  }
+}
+```
```
Comment (LBHakairamkonda) [inline README.md]: Response should be as formatted below:
   {
   "results": [
    {
	  "rentAccountNumber": "0123456",
      "parisReference": "00032896",
      "postcode": "AB1 1AA",
      "address": "Ross Court 2"
    },
	{
	  "rentAccountNumber": "0123456",
      "parisReference": "00032896",
      "postcode": "AB1 1AA",
      "address": "Ross Court 2"
    },
	{
	  "rentAccountNumber": "0123456",
      "parisReference": "00032896",
      "postcode": "AB1 1AA",
      "address": "Ross Court 2"
    }
  ]
}

**Review by LBHakairamkonda — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -443,4 +443,30 @@ A successful response should create the Work order in DRS and get the list of av
   ]
 }
 ```
+## Get account and address for housing residents
 
+Returns a list of account and address information
+
+```
+Get /v1/CRMAccounts?parisReference=221234470&postcode=E8 1HH
```
Comment (LBHakairamkonda) [inline README.md]: url should be always lowercase, could you please modify this?

File: README.md [hunk 3]
```diff
@@ -443,4 +443,30 @@ A successful response should create the Work order in DRS and get the list of av
...
+- Postcode (required)
+
+### Response
+A successful response should get a list of account and address information corresponding to the required parameters.
+
+```json
+
+ {
+ "results": [
+		  {
```
Comment (LBHakairamkonda) [inline README.md]: Are you returning the account number as well? If yes, please add to the response.

**Review by LBHakairamkonda — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (MirelaGeorgiou) [reply README.md]: No, we are not returning the account number. The response I have added is the actual response that the API returns.

**Review by LBHakairamkonda — APPROVED** [review]

**Pull request merged**